realsense2_camera/src/realsense_node_factory.cpp

```diff
@@ -270,7 +270,7 @@ void RealSenseNodeFactory::onInit()
 	auto privateNh = getPrivateNodeHandle();
 
 	privateNh.param("initial_reset", _initial_reset, false);
-	_init_timer = nh.createWallTimer(ros::WallDuration(1.0), &RealSenseNodeFactory::initialize, this, true);
+	_init_timer = nh.createWallTimer(ros::WallDuration(0.1), &RealSenseNodeFactory::initialize, this, true);
```

Comment:
IIUC this changes the timer's callback from 1 sec to 0.1 sec? No idea if that's acceptable in the upstream (but maybe you have a rationale @malban). Let's hear what the maintainers say.

Reply:
This timer is new, so I don't thing the change in duration will matter to upstream.  The duration isn't important.  It's just to separate the driver initialization from the onInit() function, which shouldn't block or do significant work: http://wiki.ros.org/nodelet#onInit

In my experience, if onInit takes too long, nodelets can silently fail to correctly advertise/subscribe to topics.